addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/client/XmlRpcClient.java

```diff
@@ -47,13 +46,6 @@ public XmlRpcClient(HomematicConfig config) throws IOException {
 
     @Override
     public void dispose() {
```

Comment:
Is this method needed anymore? If not you  an remove it completely.

Reply:
The `BinRpcClient` implementation uses java Sockets instead of the jetty client. Therefore, the dispose() method is still needed in the `RpcClient` superclass and must also be overridden by the `XmlRpcClient`.

One alternative would be to put the empty method body into the `RpcClient` instead, which would get rid of the method in the `XmlRpcClient`, but I am not sure whether this would be a better solution.

Reply:
Ok, it makes sense to leave it this way. I somehow missed that this method is an implementation for the abstract method of the superclass.